runtime/testy/Ansible.py

```diff
@@ -0,0 +1,98 @@
+import socket
+import threading
+import time
+import fake_dawn
+import random
+import sys
+from runtimeUtil import *
+data = [0] #for testing purposes
+sendPort = 1235
+recvPort = 1236
+
+#Custom buffer for handling states. Holds two states, updates one and sends the other.
+class two_buffer(): 
```

Comment:
Name this `TwoBuffer` so that it follows the proper naming convention for classes. (You can consult with PEP8 for style advice)